en/reference/annotations-mapping.rst

```diff
@@ -50,7 +50,9 @@ Optional attributes:
 
 -  **nodeType**: PHPCR type for this node, default ``nt:unstructured``.
 -  **repositoryClass**: Name of the repository to use for this document.
--  **versionable**: Set to true to enable versioning, implies *referenceable*.
+-  **versionable**: Set to ``"simple"`` or ``"full"`` to enable versioning 
+   (respectively simple or full level). Set to ``"false"`` to disable versioning
```

Comment:
Bu this logic however we should change all string literals to

````
`"string"`
````

What about if we just say

````
**versionabl** (string) Set to ...
````

I think there are already examples of this